### ISSUE TITLE
docs: Update printing infrastructure docs

### ISSUE THREAD

**Issue by abizer** [ocf/ocfweb#529]
with https://github.com/ocf/puppet/pull/684 we should update https://github.com/ocf/ocfweb/blob/master/ocfweb/docs/docs/staff/backend/printhost.md to reflect the new architecture.

**Comment by jxlin123** [comment]
Agreed. Do you think I should also make another page to describe the local CUPS printing queues, or should it all be contained within the document we currently have? Just wondering since now the printing process doesn't solely reside on printhost anymore

**Comment by dkess** [comment]
I'd keep it all in one page.

**Comment by cg505** [comment]
We also need to update https://www.ocf.berkeley.edu/docs/staff/scripts/pdf-open/
